core/src/main/java/com/qq/tars/protocol/tars/support/TarsPromiseFutureCallback.java

```diff
@@ -0,0 +1,99 @@
+package com.qq.tars.protocol.tars.support;
+
+
+import com.qq.tars.client.ServantProxyConfig;
+import com.qq.tars.client.cluster.ServantnvokerAliveChecker;
+import com.qq.tars.client.rpc.tars.TarsInvoker;
+import com.qq.tars.client.util.ClientLogger;
+import com.qq.tars.common.util.Constants;
+import com.qq.tars.net.client.Callback;
+import com.qq.tars.protocol.util.TarsHelper;
+import com.qq.tars.rpc.exc.ServerException;
+import com.qq.tars.rpc.exc.TarsException;
+import com.qq.tars.rpc.protocol.tars.TarsServantResponse;
+import com.qq.tars.support.stat.InvokeStatHelper;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * jce的CompletableFuture的实现
+ * @author Kerrigan
```

Comment:
Most of open source projects have checkstyle rule to forbid `@author` tag. It is because we can always track this information with version control tool(here is Git). See also [this article](https://www.vojtechruzicka.com/stop-using-javadoc-author-tag/).